ui/src/app/pages/analysis/notebook-list.tsx

```diff
@@ -58,8 +59,10 @@ export const NotebookList = withCurrentWorkspace()(class extends React.Component
     try {
       const {workspace: {namespace, id}} = this.props;
       this.setState({loading: true});
-      const notebooks = await workspacesApi().getNoteBookList(namespace, id);
-      this.setState({notebooks});
+      const notebookList = await workspacesApi().getNoteBookList(namespace, id);
+      this.setState({notebookList});
+      const notebookNameList = notebookList.map(fd => fd.name.slice(0, -6)); // remove '.ipynb'
```

Comment:
Is the suffix the same for R notebooks?

Reply:
good point! I just checked and it is the same suffix.

r-notebook-name.ipynb

Reply:
Tip: You can get the same documentation without the comment like so:
```javascript
fd.name.slice(0, -('.ipynb'.length))
```